lib/repository.py

```diff
@@ -124,25 +104,23 @@ def _update_submodules(self):
         Update repository submodules, initializing them if needed.
         """
         cmd = "git submodule init; git submodule update"
```

Comment:
Yep. I'll add a TODO and do it in another patch, along with archive.
This patch should be only about removing pygit2.